### PR TITLE
Update Obsidian Weathers - TrueStorms Merged Compatibility

### PR DESCRIPTION
**Pull request by ddbb07**
- Removed unecessary incompatible message for "Obsidian Weathers.esp"
- Fixed patch messages
- Removed clean messages from patches

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -3190,16 +3190,21 @@ plugins:
...
-        condition: 'active("TrueStormsSE.esp") and not active("Audio Overhaul Skyrim - True Storms Patch.esp") and not active("(Aequinoctium|ELFXWeathers|Mythical|NAT|Obsidian|Rustic|Vivid)_TS_Patch_.*\.esp")'
+        condition: 'active("TrueStormsSE.esp") and not active("Audio Overhaul Skyrim - True Storms Patch.esp") and not active("Obsidian Weathers.esp") and not active("(Aequinoctium|ELFXWeathers|Mythical|NAT|Obsidian|Rustic|Vivid)_TS_Patch_.*\.esp")'
       - <<: *patchProvided
         subs: [ 'Vivid Weathers Definitive Edition' ]
         condition: 'active("Vivid WeathersSE.esp") and not active("Audio Overhaul Skyrim - Vivid Weathers Patch.esp") and not active("Vivid Weathers SE - AOS Patch.esp") and not active("Vivid_TS_Patch_.*\.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - 'True Storms and Obsidian Weathers'
+          - '[Obsidian Weathers - TrueStorms Merged Compatibility - AOS Patch](https://www.nexusmods.com/skyrimspecialedition/mods/17198/)'
+        condition: 'active("TrueStormsSE.esp") and active("Obsidian Weathers.esp") and not active("Obsidian_TS_AOS_Patch.esp") and not active("True Storms - Obsidian Weathers - Patch .esp")'
```
Comment (MacSplody) [inline masterlist.yaml:3207]: This is duplicating messages, it's already being shown in the patch metadata.

```yaml
  # Obsidian Weathers & True Storms Merged Compatibility Patch
  - name: 'Obsidian_TS_Patch_(Luminous|Spectral)(_WaterFix)?\.esp'
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/17198/' ]
    inc:
      - 'Audio Overhaul Skyrim - Obsidian Weathers Patch.esp'
      - 'Audio Overhaul Skyrim - True Storms Patch.esp'
      - 'SoS_Obsidian_Patch.esp'
      - 'SoS_TrueStorms_Patch.esp'
      - 'True Storms - Obsidian Weathers - Patch .esp'
    tag: [ Sound ]
    msg:
      - <<: *patchIncluded
        subs: [ 'Audio Overhaul' ]
        condition: 'active("Audio Overhaul Skyrim.esp") and not active("Obsidian_TS_AOS_Patch.esp")'
      - <<: *patchIncluded
        subs: [ 'Sounds of Skyrim Complete' ]
        condition: 'active("SoundsofSkyrimComplete.esp") and not active("Obsidian_TS_SoS_Patch.esp")'
      - <<: *useOnlyOneX
        subs: [ 'Obsidian Weathers - TrueStorms Merged Compatibility' ]
        condition: 'many("Obsidian_TS_Patch_(Luminous|Spectral)(_WaterFix)?\.esp")'
```

File: masterlist.yaml [hunk 2]
```diff
@@ -3190,16 +3190,21 @@ plugins:
...
-        condition: 'active("Obsidian Weathers.esp") and not active("Audio Overhaul Skyrim - Obsidian Weathers Patch.esp") and not active("Obsidian_TS_Patch__.*\.esp")'
+        condition: 'active("Obsidian Weathers.esp") and not active("Audio Overhaul Skyrim - Obsidian Weathers Patch.esp") and not active("TrueStormsSE.esp") and not active("Obsidian_TS_Patch__.*\.esp")'
       - <<: *patchProvided
         subs: [ 'Rustic Weathers and Lighting' ]
         condition: 'active("Rustic Weathers And Lighting.esp") and not active("Audio Overhaul Skyrim - Rustic Weathers Patch.esp") and not active("Rustic_TS_Patch_.*\.esp")'
       - <<: *patchProvided
         subs: [ 'True Storms - Thunder, Rain & Weather Redone' ]
-        condition: 'active("TrueStormsSE.esp") and not active("Audio Overhaul Skyrim - True Storms Patch.esp") and not active("(Aequinoctium|ELFXWeathers|Mythical|NAT|Obsidian|Rustic|Vivid)_TS_Patch_.*\.esp")'
+        condition: 'active("TrueStormsSE.esp") and not active("Audio Overhaul Skyrim - True Storms Patch.esp") and not active("Obsidian Weathers.esp") and not active("(Aequinoctium|ELFXWeathers|Mythical|NAT|Obsidian|Rustic|Vivid)_TS_Patch_.*\.esp")'
```
Comment (MacSplody) [inline masterlist.yaml:3199]: If I recall correctly you can use Obsidian, AOS and True Storm together with the official patches.
Though you may still need to do some of your own patches for it to work 100%.
So I don't see why we should be hiding messages if the official patches work together.

File: masterlist.yaml [hunk 3]
```diff
@@ -3823,6 +3819,11 @@ plugins:
           - 'Mythical Ages'
           - '[Mythical Ages - TrueStorms Merged Compatibility](https://www.nexusmods.com/skyrimspecialedition/mods/15617/)'
         condition: 'active("Mythical Ages.esp") and not active("Mythical_TS_Patch_.*\.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - 'Obsidian Weathers and Seasons'
+          - '[Obsidian Weathers - TrueStorms Merged Compatibility](https://www.nexusmods.com/skyrimspecialedition/mods/17198/)'
+        condition: 'active("Obsidian Weathers.esp") and not active("Obsidian_TS_Patch_.*\.esp") and not active("True Storms - Obsidian Weathers - Patch .esp") and not file("Wander.esp")'
```
Comment (MacSplody) [inline masterlist.yaml:3826]: This is also a duplicate, but I think moving it from `Obsidian Weathers.esp`'s metadata for consistency is fine.
So you'd need to remove that duplicate message from `Obsidian Weathers.esp`'s metadata.

File: masterlist.yaml [hunk 4]
```diff
@@ -3741,9 +3737,9 @@ plugins:
       - 'Obsidian_TS_Patch_Luminous_WaterFix.esp'
       - 'Obsidian_TS_Patch_Spectral_WaterFix.esp '
       - 'Obsidian Weathers - Wander Patch.esp'
-    clean:
-      - crc: 0x795E49B4
-        util: 'SSEEdit v4.0.3'
+    msg:
+      - <<: *useInstead
+        subs: [ '[Obsidian Weathers - TrueStorms Merged Compatibility](https://www.nexusmods.com/skyrimspecialedition/mods/17198/)' ]
```
Comment (MacSplody) [inline masterlist.yaml:3742]: I can't really agree with this, because either set of patches should work.
Has there been some major update for any of these mods lately?

**Comment by ddbb07** [comment]
I decided to close this
